spec/latest/core/error.jsonld

```diff
@@ -0,0 +1,11 @@
+{
+  "@context": {
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "hydra": "http://www.w3.org/ns/hydra/core#",
+    "type": "@type",
```

Comment:
```suggestion
    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
    "type": { "@id": "rdf:type", "@type": "@id" },
```

As suggested by @lanthaler, this makes this context usable by JSON-LD 1.0 processors